src/ResourceManagement/AzureBackup/RecoveryServices.Tests/ScenarioTests/RecoveryServicesTestBase.cs

```diff
@@ -0,0 +1,114 @@
+﻿//
+// Copyright (c) Microsoft.  All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+using System;
+using System.Configuration;
+using System.Net;
+using System.Net.Http;
+using System.Net.Security;
+using System.Reflection;
+using Microsoft.Azure;
+using Microsoft.Azure.Management.RecoveryServices.Backup;
+using Microsoft.Azure.Management.RecoveryServices.Backup.Models;
+using Microsoft.Azure.Test;
+using Microsoft.Azure.Test.HttpRecorder;
+
+namespace RecoveryServices.Tests
+{
+    public class RecoveryServicesTestsBase : TestBase
+    {
+        public static T GetServiceClient<T>(string resourceNamespace) where T : class
```

Comment:
do you want to duplicate all these methods across backupvault and RSvault? How about creating some common which both projects reference